B827EBFFFE937A49.json

```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "eui-b827ebfffe937a49", 
+		"servers": [ 
+			{ 
+				"server_address": "asia-se.thethings.network", 
```

Comment:
This is not the router address, please check here for the list of valid addresses: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses

(I guess you want `router.as1.thethings.network` or `router.as2.thethings.network`)

Reply:
ok thanks

Reply:
"server_address": "router.as1.thethings.network",